roamance/resource/roamance-srs-client.md

```
@@ -0,0 +1,160 @@
+- Set travel goals
+- Track travel stats (countries visited, miles traveled)
+
+#### 1.14 Posts
+- Create and share posts with text, images, or videos
+- Tag destinations, activities, or fellow travelers
+- Interact with posts (like, comment, share)
+- Bookmark posts for future reference
+
+### 2. AI-Powered Frontend Features
+
+#### 2.1 AI Itinerary Generator
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)